### ISSUE TITLE
yapfify-buffer doesn't format buffer if a mark has never been set

### ISSUE THREAD

**Issue by volnt** [JorisE/yapfify.el#10]
Whenever I open a new Python buffer I need to set the mark and deactivate it at least once otherwise `yapfify-buffer` crashes with `Error: (error "The mark is not set now, so there is no region")`.

If I set & deactivate it at least once then everything works as expected.

```
/tmp $ emacsclient --version
emacsclient 24.5
/tmp $ emacs --version
GNU Emacs 24.5.1
```

I made a recording showing the bug, you can see in the `*Messages*` buffer on the right what's happening on the left : 

* I create a file /tmp/test.py with `hello="world"` expecting that yapfify will reformat it to `hello = "world"`
* I save the file, it prints the error `Error: (error "The mark is not set now, so there is no region")` and doesn't reformat the file
* I set the mark and deactivate it (C-<SPC> + C-<SPC>)
* I save the file, it reformats the file as expected

![tty](https://user-images.githubusercontent.com/2411932/33654490-94cb730c-da70-11e7-8cc0-40b95eaa8e7a.gif)

**Comment by JorisE** [comment]
Thanks for reporting, I think I see where this went wrong.

**Comment by proofit404** [comment]
Hi, thanks for reporting!

Should be fixed by 620f345d87510b003cec1eca14c0938ababd8875
